src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/config/ErrorConfiguration.kt

```diff
@@ -94,16 +96,33 @@ class ErrorConfiguration(private val telemetryClient: TelemetryClient) {
     return errorResponse(HttpStatus.CONFLICT, "entity already exists", e.message)
   }
 
+  @ExceptionHandler(WebClientRequestException::class)
+  fun handleWebClientRequestException(e: WebClientRequestException): ResponseEntity<ErrorResponse> {
+    logger.info("web client request exception", e)
+    return errorResponse(HttpStatus.INTERNAL_SERVER_ERROR, "web client request exception", e.message, userMessageForWebClientException(HttpStatus.INTERNAL_SERVER_ERROR))
+  }
+
   @ExceptionHandler(WebClientResponseException::class)
   fun handleWebClientResponseException(e: WebClientResponseException): ResponseEntity<ErrorResponse> {
-    logger.info("web client exception", e)
-    return errorResponse(e.statusCode, "web client exception", e.responseBodyAsString)
+    logger.info("web client response exception", e)
+    return errorResponse(e.statusCode, "web client response exception", e.responseBodyAsString, userMessageForWebClientException(e.statusCode))
```

Comment:
If we're adding user messages, we should also rename "web client ... exception" to "call to a dependency failed", to help explain what is happening

Reply:
will do